csc-services/services/ShipmentsService.js

```diff
@@ -54,6 +63,26 @@ class ShipmentsService extends DSUService {
       shipment.uid
     );
 
+    this.sendMessageToEntity(
+      CommunicationService.identities.CSC.SPONSOR_IDENTITY,
+      messagesEnum.ShipmentInPreparation,
+      {
+        orderSSI: order.uid,
+      },
+      'Shipment In Preparation'
```

Comment:
I see you have added a constant with this String. Please use it.

Reply:
Update done